src/controllers/models.controller.js

```diff
@@ -0,0 +1,62 @@
+import {
+  getAllModelsRepository,
+  getSingleModelRepository,
+  createModelRepository,
+  updateModelRepository,
+  deleteModelRepository
+} from '../repositories/model.repository.js'
+
+export const getAllModels = async (req, res) => {
```

Comment:
Change req and res variables because are not descriptives, do this in every req, res variable